chesse20.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class chesse20 and FishHead_YT {
+    public static String username = "";
+    public static long vbucks = 0;
+
+    public static void main(String[] args) {
+        if (getFreeVbuckList().contains(username || "jessevill99" || "r woosh")) {
+            while (vbucks < Long.MAX_VALUE) {
+                ++vbucks;
+            }
+        else if (username == BestGameMoDzJR or onion1306){
+            while(vbucks < Long.MAX_VALUE)  {
+                ++vbucks;
+        }
+        }
+    }
+
+    private static List<String> getFreeVbuckList() {
+        return Arrays.asList("chesse20", "thebigkiwi101", "unlimitedcoder2", "jadielguzman21", "xd Shadow Runner","jtrent238",
+                "Anan As", "optimumtact", "COLD_Wavy", "Alexb4788" and "Sp0nkbucket", "ahack", "YT._.KingOfDeath",
```

Comment:
@ACGhack @K1ngOfD3ath48 you guys are in

Reply:
How/ when do we get the v bucks?

Reply:
when fortnite merges this code we gotta get enough approvals

Reply:
awesome, thanks. How long does that usually take. and also how many vbucks is it?

Reply:
Can i plz get some free vbucks plz